### ISSUE TITLE
snipper-chosser.js may be incompatible with wagtail 2.11.1

### ISSUE THREAD

**Issue by brookzhcn** [wagtail/wagtail-react-streamfield#68]
I upgraded wagtail to 2.11.1 and found it can't open snipper chooser modal in admin.I found that snipper-chosser.js is not same with the official one.  This could be a upgrade isssue.

**Comment by murray3k** [comment]
Yes wagtail changed where the snipper chooser gets it URL from. I provided a small fix for this see: #69 .

**Comment by roodie** [comment]
With the latest wagtail version (2.12) the page-chooser.js also causing errors unfortunately.
